style.css

```diff
@@ -1,3 +1,3 @@
 h1 {
-  color: rgb(212, 76, 76);
+  color: rgb(122, 15, 15);
 }
```

Comment:
- [ ] Please try to use one class with styles for your header instead of using `h1` tag.

Reply:
ok thanks noted